iwyu.cc

```diff
@@ -4034,6 +4057,16 @@ class IwyuAstConsumer
     return Base::VisitUsingDirectiveDecl(decl);
   }
 
+  bool AllTypesProvided(const Type* typedef_underlying_type) const {
```

Comment:
I'd prefer it if this took the location as a parameter as well, to avoid the implicit dependency on `current_ast_node()`.